tests/test__routines.py

```diff
@@ -10,7 +10,8 @@
 
 @pytest.mark.parametrize("use_threads", [True, False])
 @pytest.mark.parametrize("concurrent_partitioning", [True, False])
-def test_routine_0(glue_database, glue_table, path, use_threads, concurrent_partitioning):
+@pytest.mark.parametrize("table_type", ["EXTERNAL_TABLE", "GOVERNED"])
```

Comment:
Test routine has been tweaked to allow for "GOVERNED" tables as a test parameter

Reply:
This test case is tough!